src/pages/releases.mdx

```diff
@@ -65,7 +65,7 @@ MySQL requires the use of the [mysql2] npm package.
 
 | Sequelize   | [MySQL][mysql] | [mysql2] |
 |-------------|----------------|----------|
-| 7 (alpha)   | ^5.7, ^8.0     | >= 2.3.3 |
+| 7 (alpha)   | >=8.0.19       | >= 3.9.1 |
```

Comment:
Not sure about the adapter change, earlier versions should work but we just don't test on it

Reply:
Due to the import in the link below, the older version supported is [3.8.0](https://github.com/sidorares/node-mysql2/releases/tag/v3.8.0)

https://github.com/sequelize/sequelize/blob/3b8349586a24e92f15c1f5f4d011b44dcf9235b7/packages/core/src/dialects/mysql/connection-manager.ts#L9

Reply:
Ah yes, that's true. When the user is using TypeScript we'll need 3.8.0